x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/analytics/AnalyticsCollectionService.java

```diff
@@ -124,7 +129,10 @@ public void deleteAnalyticsCollection(
         assert (state.nodes().isLocalNodeElectedMaster());
 
         AnalyticsCollection collection = new AnalyticsCollection(request.getCollectionName());
-        DeleteDataStreamAction.Request deleteDataStreamRequest = new DeleteDataStreamAction.Request(collection.getEventDataStream());
+        DeleteDataStreamAction.Request deleteDataStreamRequest = new DeleteDataStreamAction.Request(
+            TimeValue.THIRTY_SECONDS /* TODO should we wait longer? */,
```

Comment:
Unsure about this timeout tbh, I'm leaving it at 30s but at least calling it out as such here.